### ISSUE TITLE
Calibration Cache Unnamed Layer for ONNX & Caffe Models

### ISSUE THREAD

**Issue by Hassan313** [NVIDIA/TensorRT#681]
I am Getting Unnamed Layers in the calibration cache file. I have used both ONNX model and Caffe model which has created Unnamed Layer. These layers are "batch normalization", "scale" and "Relu". 

TRT-7000-EntropyCalibration2
data: 3caa54fc
(Unnamed Layer* 0) [Convolution]_output: 3d6969df
(Unnamed Layer* 1) [Scale]_output: 3ac9c5e4
(Unnamed Layer* 2) [Scale]_output: 3bb23f5c
conv1: 3bb2132a
pool1: 3bb2132a
(Unnamed Layer* 5) [Convolution]_output: 3b0f3403
(Unnamed Layer* 6) [Scale]_output: 3c19f4a8
res2a_branch1: 3b39bf9e
(Unnamed Layer* 8) [Convolution]_output: 3b64b387
(Unnamed Layer* 9) [Scale]_output: 3c34ac95
(Unnamed Layer* 10) [Scale]_output: 3b309597
res2a_branch2a: 3b21cb45
(Unnamed Layer* 12) [Convolution]_output: 3acc6b2e
(Unnamed Layer* 13) [Scale]_output: 3cb1a784
res2a_branch2b: 3b5eba4f
(Unnamed Layer* 15) [ElementWise]_output: 3ba75a96
res2a: 3b73bec7
(Unnamed Layer* 17) [Convolution]_output: 3b8658ae
(Unnamed Layer* 18) [Scale]_output: 3cce329f
(Unnamed Layer* 19) [Scale]_output: 3b9b34cf
res2b_branch2a: 3b6795f7
(Unnamed Layer* 21) [Convolution]_output: 3b1fb3bc
(Unnamed Layer* 22) [Scale]_output: 3d1fdc85
res2b_branch2b: 3c0d6bd4
(Unnamed Layer* 24) [ElementWise]_output: 3ba9e0a1
res2b: 3ba9e0a1
(Unnamed Layer* 26) [Convolution]_output: 3b16170a
(Unnamed Layer* 27) [Scale]_output: 3cf6451c
res3a_branch1: 3b263b66
(Unnamed Layer* 29) [Convolution]_output: 3ba9a8f5
(Unnamed Layer* 30) [Scale]_output: 3cc67d8a
(Unnamed Layer* 31) [Scale]_output: 3ba7f0eb
res3a_branch2a: 3b86b13f
(Unnamed Layer* 33) [Convolution]_output: 3ae5e952

## Environment

**TensorRT Version**: 7
**GPU Type**: Ti
**Nvidia Driver Version**: 
**CUDA Version**: 10
**CUDNN Version**: 7
**Operating System + Version**: Ubuntu 18
**Python Version (if applicable)**: 3.6
**TensorFlow Version (if applicable)**: 
**PyTorch Version (if applicable)**: 
**Baremetal or Container (if container which image + tag)**:

**Comment by rmccorm4** [comment]
Issue also being tracked here: https://forums.developer.nvidia.com/t/calibration-cache-unnamed-layer/141610

**Comment by nvpohanh** [comment]
@rmccorm4 Is this causing any issue? By default, TRT gives tensors names like `(Unnamed Layer* 0) [Convolution]_output` unless you manually override it using TRT APIs.

**Comment by rmccorm4** [comment]
Hi @nvpohanh , more of a quality of life issue than a blocker. However since the issue is so old, I think we can just close it for now.
